game/src/logging/CustomLogLevel.java

```diff
@@ -0,0 +1,14 @@
+package logging;
+
+import java.util.logging.Level;
+
```

Comment:
Javadoc missing

Reply:
hinzugefügt in 14a44b47578812e63dc84483d24d1fcb972e6980